specifyweb/frontend/js_src/lib/components/DataModel/businessRuleDefs.ts

```diff
@@ -202,6 +202,20 @@ export const businessRuleDefs: MappedBusinessRuleDefs = {
             });
         return undefined;
       },
+      parentCog: async (cog): Promise<BusinessRuleResult> => {
+        if (cog.url() === cog.get('parentCog')) {
+          return {
+            isValid: false,
+            reason: resourcesText.parentCogSameAsChild(),
+            saveBlockerKey: PARENTCOG_KEY,
+          };
+        }
+
+        return {
+          isValid: true,
+          saveBlockerKey: PARENTCOG_KEY,
+        };
+      },
```

Comment:
Do we need this business rule at all? The `parentCog` business rule in COG helps avoid linking a COG to itself - but that will not be a problem in a CO